src/renderTranslatedRoutes.js

```diff
@@ -75,7 +68,6 @@ const renderTranslatedRoutes = (locales, routes, pathFromRouteForPathsAndLocale)
           pathFromRouteForPathsAndLocale,
         ),
         renderTranslatedRoutesForCurrentConfig = renderTranslatedRoutesForConfig(
-          configRoute,
```

Comment:
I would move this changes to another PR. 

Changes look great but I would add tests for `renderTranslatedRoutes` to ensure we are not breaking anything